src/components/Stake/CardAccordion.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonIcon,
+  Card,
+  GU,
+  IconUp,
+  IconDown,
+  RADIUS,
+  useTheme,
+} from '@aragon/ui'
+import { Spring, Transition, animated } from 'react-spring/renderprops'
+import styled from 'styled-components'
+
+const CardAccordion = React.memo(function CardAccordion({ card, expansion }) {
+  const [opened, setOpened] = useState(false)
+  const theme = useTheme()
+
+  const toggleButton = useCallback((index) => {
+    setOpened((opened) => !opened)
+  }, [])
+
+  return (
+    <div
+      css={`
+        position: relative;
+        margin-bottom: ${RADIUS}px;
+      `}
+    >
+      <OpenedSurfaceBorder opened={opened} />
+      <Card
+        css={`
+          width: 100%;
+          z-index: 2;
+        `}
+      >
```

Comment:
At the moment these cards seem to have a fixed height. I was thinking it would be better to set it to `auto` and use padding to wrap the inner content. We could then aim to match the height in the design:

![image](https://user-images.githubusercontent.com/11708259/92503312-fb636500-f1f8-11ea-869e-6bc6011605c9.png)